tests/PackageFilterTest.php

```diff
@@ -52,7 +52,7 @@ public function testRemoveLegacyPackages(array $expected, array $packages, strin
         $expected = $configToPackage($expected);
         $packages = $configToPackage($packages);
 
-        $actual = $filter->removeLegacyPackages($packages);
+        $actual = $filter->removeLegacyPackages($packages, []);
```

Comment:
We should have a test covering the non-empty array.

Reply:
test added